tox.ini

```
@@ -0,0 +1,22 @@
+[tox]
+envlist=lint,py{27,36}
+
+[testenv]
+install_command=pip install -e '.[testing]' {packages}
+commands=
+    coverage erase
+    coverage run --source='cfgov_setup' -m unittest cfgov_setup.tests {posargs}
+    coverage report -m
+
+basepython=
+    py27: python2.7
+    py36: python3.6
+
+[testenv:lint]
+basepython=python3.6
+deps=
+    flake8>=2.2.0
+    isort>=4.2.15
+commands=
+    flake8 cfgov_setup
+    isort --check-only --diff cfgov_setup
```

This should do the trick:

```suggestion
    isort --check-only --diff --recursive cfgov_setup
```